--- modules/appservice/main.tf.orig	2023-07-27 15:07:34.484226000 +1200
+++ modules/appservice/main.tf	2023-07-27 15:08:44.004579500 +1200
@@ -128,6 +128,17 @@
       remote_debugging_version = lookup(site_config.value, "remote_debugging_version", null)
       scm_type                 = lookup(site_config.value, "scm_type", null)
       websockets_enabled       = lookup(site_config.value, "websockets_enabled", null)
+
+      dynamic "application_stack" {
+        for_each = lookup(site_config.value, "application_stack", null) != null ? [lookup(site_config.value, "application_stack", {})] : []
+
+        content {
+          docker_image_name = lookup(application_stack.value, "docker_image_name", null)
+          docker_registry_url = lookup(application_stack.value, "docker_registry_url", null)
+          docker_registry_username = lookup(application_stack.value, "docker_registry_username", null)
+          docker_registry_password = lookup(application_stack.value, "docker_registry_password", null)
+        }
+      }
     }
   }
 
@@ -183,6 +194,17 @@
       remote_debugging_version = lookup(site_config.value, "remote_debugging_version", null)
       scm_type                 = lookup(site_config.value, "scm_type", null)
       websockets_enabled       = lookup(site_config.value, "websockets_enabled", null)
+
+      dynamic "application_stack" {
+        for_each = lookup(site_config.value, "application_stack", null) != null ? [lookup(site_config.value, "application_stack", {})] : []
+
+        content {
+          docker_image_name = lookup(application_stack.value, "docker_image_name", null)
+          docker_registry_url = lookup(application_stack.value, "docker_registry_url", null)
+          docker_registry_username = lookup(application_stack.value, "docker_registry_username", null)
+          docker_registry_password = lookup(application_stack.value, "docker_registry_password", null)
+        }
+      }
     }
   }
 
